docs/FAQ/Accounts.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Accounts & Management
+parent: FAQ
+nav-order: 1
+---
+# Accounts
+
+> Q: Do I need an account?
+>
+> A: An account is not really needed to play on [Zombies](ServernGamemodes.md#Zombies), [PvE](ServernGamemodes.md#PvE), [PvP](ServernGamemodes.md#PvP), [etc](ServernGamemodes.md), **BUT** it is required for [Survival](ServernGamemodes.md#Survival)
+
+## Registering
+
+> Q: How do I register?
+>
+> A: To register an account you **MUST** be in rift(`/rift`), do `/register <password>`, then do `/login <password>`.
+
+> Q: `/register` says I already own an account.
+>
+> A: Aslong as you have logged in **ATLEAST** 1 time into your account, you can simply do `/rename <name the /register command told you>` then `/login` without password(this is of course assuming you have the same IP and Computer), but if you haven't already logged in atleast one time, do `/login <password>` instead of `/login` without a password.
```

Comment:
```suggestion
> A: As long as you have logged in **AT LEAST** once into your account, you can simply do `/rename <name the /register command told you>` then `/login` without password (this is of course assuming you have the same IP and Computer), but if you haven't already logged in at least once, do `/login <password>` instead of `/login` without a password.
```